app/javascript/multi-file-upload.js

```diff
@@ -12,32 +12,31 @@ MOJFrontend.MultiFileUpload.prototype.uploadFiles = async function(files) {
   const uploads = []
 
   for(const file of files) {
-    uploads.push(
-      this.uploadFile(file).then(
-        function(response) { console.log(response); }
-      ).catch(
-        function(error) { console.log(error);  }
-      )
-    );
+    uploads.push(this.uploadFile(file)
```

Comment:
An example of this would be if they made it so that the component only used uploadFile if you drag one file in and only used uploadFiles if you drag multiple files in - if they make that update, we'd have to move the logic back.

Reply:
My point is the `true`/`false` promise resolution I'm not convinced is doing anything. If it is for some reason needed for the error handling to work then fine, but if it's not doing anything I'd rather it just be removed

Reply:
ah ok got you - no it currently doesn't do anything, just makes it so that it _could_ do something if we wanted to. But yeah you're right no need to have it there if it doesn't do anything. 

Will just not do anything in both scenarios

Reply:
I thought you were removing the chain?